functions.js

```diff
@@ -5,7 +5,6 @@
 // -1 - the guessed number is too low
 // 1 - the guessed number is too high
 export const compareNumbers = (guess, correctNumber = 7) => {
```

Comment:
Since compareNumbers() takes secretNumber as an argument in your app.js, it would be best for you to change your correctNumber to not equal anything. Basically you don't need the added '= 7'. 
Nice job on the randomly generated number!

Reply:
good point, I added this for purposes of running the test.  I tested against the default value.  Not sure if this is a best practice though.